packages/api/src/typeDefs/query.graphql

```diff
@@ -44,6 +44,14 @@ type StoreSearchResult {
   facets: [StoreFacet!]!
 }
 
+type StorePerson {
+  id: String
```

Comment:
This is bugging me. Shouldn't `id`, alongside `email` and other fields be required, like in `id: String!`. `email: String!` etc?